schutzbot/ci_details.sh

```diff
@@ -25,4 +25,7 @@ CI MACHINE SPECS
 
 ------------------------------------------------------------------------------
 EOF
-echo -e "\033[0m"
\ No newline at end of file
+echo -e "\033[0m"
+
+rpm -qa | sort
```

Comment:
Can you add something like this?

```suggestion
echo List of installed packages:
rpm -qa | sort
```